h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/predict/XGBoostJavaBigScorePredictTest.java

```diff
@@ -0,0 +1,93 @@
+package hex.tree.xgboost.predict;
+
+import biz.k11i.xgboost.Predictor;
+import hex.genmodel.algos.tree.SharedTreeGraph;
+import hex.genmodel.algos.tree.SharedTreeNode;
+import hex.tree.xgboost.XGBoostModel;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.runner.H2ORunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(H2ORunner.class)
+public class XGBoostJavaBigScorePredictTest extends TestUtil  {
+
+    @Test
+    public void testFindUsedColumns_few() {
+        XGBoostModel.XGBoostParameters parms = new XGBoostModel.XGBoostParameters();
+        parms._weights_column = "CAPSULE";
+
+        boolean[] used = checkFindUsedColumns(parms, 5);
+        assertTrue(countTrues(used) > 0);
```

Comment:
Is there any logical explanation why these two conditions must hold?

Reply:
I improved the test to be more clear, please let me know if it makes sense this way now.